sys-account/server/authn/pkg/auth/token.go

```diff
@@ -0,0 +1,118 @@
+package auth
+
+import (
+	"github.com/getcouragenow/packages/sys-account/authn/pkg/utilities"
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/getcouragenow/packages/sys-account/rpc"
+)
+
+var (
```

Comment:
all of these below should be parameterized as well.

Reply:
Regarding the secrets, because we need to pass a security audit and one part of that is Security at Rest, we will be encrypting the genji DB using he standard Badger encryption tools.  SO JWT_ACCESS_SECRET and JWT_REFRESH_SECRET will need to eventually be in the Genji DB.
For now leave it as it is until i get time to get the Genji encryption done.

Reply:
The JWT_ACCESS_SECRET and JWT_REFRESH_SECRET env variables are already dropped out in favor of `TokenConfig` struct which can be initialized with `accessSecret` and `refreshSecret`, following your  advice, the access and refresh secrets then can be constructed from database seed / migration(s), is that ok?